core/src/test/resources/rewrite/RedundantBraces.stat

```diff
@@ -159,3 +160,83 @@ object a {
     } // end
   }
 }
+<<< redundant enclosing braces in str interpolation
+object a {
+  s"Hello ${value}"
+}
+>>>
+object a {
+  s"Hello $value"
+}
+<<< not redundant enclosing braces in str interpolation
+object a {
+  s"Hello ${foo}bar"
+}
+>>>
+object a {
+  s"Hello ${foo}bar"
+}
+<<< enclosing braces in str interpolation
+object a {
+  s"Hello ${d.toString}"
+}
+>>>
+object a {
+  s"Hello ${d.toString}"
+}
+<<< empty str interpolation
+object a {
+  s""
+}
+>>>
+object a {
+  s""
+}
+<<< no braces in str interpolation
+object c {
+  s"${a}"
+}
+>>>
+object c {
+  s"$a"
+}
+<<< two args in str interpolation
+object c {
+  s"${a}${b}"
+}
+>>>
+object c {
+  s"$a$b"
+}
+<<< spaces before and after arg in str interpolation
+object c {
+  s" ${a} "
+}
+>>>
+object c {
+  s" ${a} "
+}
+<<< special symbol after arg no braces in str interpolation
+object c {
+  s"${a}-"
+}
+>>>
+object c {
+  s"${a}-"
```

Comment:
You can look at the spec https://www.scala-lang.org/files/archive/spec/2.11/01-lexical-syntax.html#identifiers

Seems like this rewrite should not take place when the right hand part starts with underscore or an alphanumeric character. In other cases, you can safely remove the curly brace.